.github/workflows/deploy.yml

```diff
@@ -13,15 +16,21 @@ jobs:
       contents: write
     steps:
     - name: checkout
-      uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+      uses: actions/checkout@v4
       with:
         fetch-depth: 0  # need to fetch all history to ensure correct Git revision dates in docs
 
     - name: set up Python
-      uses: actions/setup-python@13ae5bb136fac2878aff31522b9efb785519f984 # v4.3.0
+      uses: actions/setup-python@v4
       with:
         python-version: '3.10'
-
+        
+    - name: checkout test suite
+      uses: actions/checkout@v4
+      with:
+        repository: eessi/test-suite
+        path: src
```

Comment:
Why don't we just do this without path, i.e. in the current workdir? It will still clone into a subidr test-suite by default right? No need to nest that further inside a src I'd think.